### PR TITLE
Cypress test. Hotkeys to change labels feature.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add cypress test to check #3070 #3084 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            cy.get('.cvat-objects-sidebar-label-item').then(($objectsSidebarLabelItem) => {
+                firstLabelCurrentVal = $objectsSidebarLabelItem[0].innerText.slice(0, -2);
+                secondLabelCurrentVal = $objectsSidebarLabelItem[1].innerText.slice(0, -2);
+            });
+            cy.get('.cvat-objects-sidebar-tabs').within(() => {
+                cy.contains('[role="tab"]', 'Objects').click();
+            });
+        });
+
+        it('Changing a lebel for shape via hotkey.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js:72]:
```suggestion
        it('Changing a label for a shape using hotkey.', () => {
```

File: tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            cy.get('#cvat_canvas_shape_1').should('have.class', 'cvat_canvas_shape_activated');
+            cy.contains('tspan', `${firstLabelCurrentVal} 1 (manual)`).should('be.visible');
+            cy.get('body').type('{Ctrl}2')
+            cy.get('#cvat-objects-sidebar-state-item-1').find('.cvat-objects-sidebar-state-item-label-selector').should('have.text', secondLabelCurrentVal);
+            cy.contains('tspan', `${secondLabelCurrentVal} 1 (manual)`).should('be.visible');
+            // Unset settings "Always show object details"
+            testCheckingAlwaysShowObjectDetails();
+        });
+
+        it('Changing default lebel before drawing a shape.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js:99]:
```suggestion
        it('Changing default label before drawing a shape.', () => {
```

File: tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js]: Thanks. Fixed.

File: tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_72_hotkeys_change_labels.js]: Fixed.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**